dotcom-rendering/src/web/lib/dynamicSlices.tsx

```diff
@@ -70,15 +74,18 @@ export const Card75_Card25 = ({
 				containerPalette={containerPalette}
 				showAge={showAge}
 				trailText={cards[0].trailText}
+				supportingContent={cards[0].supportingContent}
```

Comment:
This can be something for another PR if it is a thing but do these need to be set in such a way that they don't both show at the same time?

Reply:
Testing this;

with trail text & supporting content set on both, the 75 card keeps both, the 25 card only has supporting content:
<img width="990" alt="image" src="https://user-images.githubusercontent.com/9575458/195639943-137d3800-3803-4597-a30c-818b985a67f3.png">

I'll look into this!

Reply:
Fixed :)